modules/log6x-creating-logfilesmetricexporter.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * observability/logging/log_collection_forwarding/cluster-logging-collector.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="log6x-creating-logfilesmetricexporter_{context}"]
+= Creating a LogFileMetricExporter resource
+
+To generate metrics from the logs produced by running containers, you must create a `LogFileMetricExporter` custom resource (CR) 
```

Comment:
```suggestion
To generate metrics from the logs produced by running containers, you must create a `LogFileMetricExporter` custom resource (CR).
```
I realize this is ported content, but I think typos and missing punctuation should be fixed.

Reply:
absolutely, fixed!